cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,83 @@
 /// <reference types='cypress' />
 
+let user;
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateNewUser').then((generateNewUser) => {
+      user = generateNewUser;
+    });
+  });
+
+  it('should provide pagination', () => {
+    cy.createNewUser(user, 5);
+
+    cy.get('select').select('5 rows');
+    cy.get('.rt-tr-group').should('have.length', '5');
+    cy.get('.-totalPages').should('contain', '2');
+
+    cy.contains('.-btn', 'Next').click();
+    cy.get('[type="number"]').should('have.value', '2');
+
+    cy.contains('.-btn', 'Previous').click();
+    cy.get('[type="number"]').should('have.value', '1');
+  });
+
+  it('should provide row selection', () => {
```

Comment:
```suggestion
  it('should provide row selection', () => {
```
this test does not have any assertions